src/components/Apps/AppItem.jsx

```diff
@@ -57,9 +58,19 @@ export class AppItem extends React.Component {
   }
 
   render() {
-    const { t, useHomeIcon, app } = this.props
+    const { useHomeIcon, app } = this.props
 
     const dataIcon = app.slug ? `icon-${app.slug}` : ''
+    const getAppDisplayName = get(
+      models,
+      'applications.getAppDisplayName',
+      app => {
+        return app.namePrefix && app.namePrefix.toLowerCase() !== 'cozy'
+          ? `${app.namePrefix} ${app.name}`
+          : app.name
+      }
+    )
```

Comment:
This declaration can be done outside of the render() method. It'll avoid the  `get` cost at every render